src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { PopupPlacement } from '../../constants';
 import { PopupBase } from '../PopupBase';
+import { TooltipAnchor } from './TooltipAnchor';
```

Comment:
Nice extraction!

Reply:
haha. thanks. It was really hard to read when they were in one file with `React.UseEffect` needing access to `setPopupVisibility`.